samples/Localization.StarterWeb/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
It didn't seem like other files in this repo had license headers so I didn't add them.